sympy/core/numbers.py

```diff
@@ -1799,9 +1799,9 @@ def _eval_power(self, expt):
                 # (3/4)**-2 -> (4/3)**2
                 ne = -expt
                 if (ne is S.One):
-                    return Rational(self.q, self.p)
+                    return Rational(self.q, self.p, 1)
                 if self.is_negative:
-                    return S.NegativeOne**expt*Rational(self.q, -self.p)**ne
+                    return S.NegativeOne**expt*Rational(self.q, -self.p, 1)**ne
```

Comment:
But here, you can't be sure that `self` was computed with `gcd=1` or not and since an evaluation is now happening it should be reduced. And is there any reason we are doing that instead of `Rational(self.q**ne, -self.p**ne, 1)`?

Reply:
Not sure about it, but my idea was that self was already reduced (since it is a `Rational`), but not fully sure.

The second rewrite is only possible if ne is integer (and some other rare cases) as the arguments to `Rational` must be integers (or `Rational`). An option is of course that `Rational(1, sqrt(2))` returns `1/sqrt(2)`, but currently this is not how it works.

Reply:
Still,  there seems to be limited benefits of adding the gcd-1 based on the timings. Although that is quite surprising considering the execution pattern of the code...

Reply:
> my idea was that self was already reduced (since it is a `Rational`),

There can be unevaluated Rationals:
```python
>>> a,b=Rational(2,4,1),Rational(3,6,1);print(a,b)
(2/4, 3/6)
>>> a**b
sqrt(2)/2
```

Reply:
> limited benefits of adding the gcd-1 based on the timings

The test may not be helpful in showing the difference, perhaps because of caching. But 
```python
got=[]
for gcd in range(2):
  for do in range(5):
    t=time.time();ok=[Rational(1,i,1) for i in range(10000)];got.append(time.time()-t)
print([sum(i) for i in got)
```
shows that there is about a 25% savings when using `gcd=1`, regardless of whether the out loop runs over `range(2)` or `reversed(range(2))`.

Reply:
Ahh, clever. Then we cannot know for sure. I'll edit.

Good that there is a gain. I was really confused as not providing the gcd seemed to execute more code, but still no gain was visible.